src/Microsoft.PowerShell.Commands.Utility/commands/utility/WriteConsoleCmdlet.cs

```diff
@@ -137,7 +137,6 @@ protected override void ProcessRecord()
             }
 
             this.WriteInformation(informationMessage, new string[] { "PSHOST" });
-            this.Host.UI.TranscribeResult(result);
```

Comment:
Have we a test for this?

Reply:
I already created three test for transcription of Write-Host command (Continue, SilentlyContinue, Ignore).
I have no idea how costly would it be to create a test for Inquire action as it makes the test interactive.

Reply:
Is it possible to add a test hook?

Reply:
Not clear where the hook should be to emulate the user interaction. Maybe in PromptForChoice method in ConsoleHostUserInterfacePromptForChoice.cs file?

Reply:
@hubuk You can add hook here https://github.com/PowerShell/PowerShell/blob/c1c5344a8897262433141ecbc13bb06ac2c4bbef/src/System.Management.Automation/engine/Utils.cs#L1492, inject in PromptForChoice and use in the test.

Reply:
That's great. Thanks.

Reply:
Tests added.